tasks/promote-buildpack/task.yml

```diff
@@ -19,6 +19,7 @@ image_resource:
   type: docker-image
   source:
     repository: pcfnorm/rootfs
+    tag: 1.0.28
```

Comment:
why are we hardcoding the rootfs version?

Reply:
See https://github.com/pivotal-cf/pcf-pipelines/pull/374/files#r234795486